fractal_server/app/schemas/user.py

```diff
@@ -52,6 +55,34 @@ class UserUpdate(schemas.BaseUserUpdate):
         val_absolute_path("cache_dir")
     )
 
+    @root_validator(pre=True)
+    def cant_set_none(cls, values):
```

Comment:
Use `validator` instead of `root_validator`, on the line of:
```python
    @validator("is_superuser", "is_superuser_2", always=True)
    def cant_set_none(cls, v):
        if v is None:
            raise ValueError("Cannot set xxx=None")
        return v
```